client/header/activity-panel/highlight-tooltip/index.js

```diff
@@ -31,22 +36,33 @@ function HighlightTooltip( {
 	onClose,
 	delay,
 	onShow = noop,
+	useAnchor = false,
 } ) {
 	const [ showHighlight, setShowHighlight ] = useState(
 		delay > 0 ? null : show
 	);
 	const [ node, setNode ] = useState( null );
+	const [ anchorRect, setAnchorRect ] = useState( null );
 
 	useEffect( () => {
 		const element = document.getElementById( id );
 		let container;
 		if ( element && ! node ) {
 			// Add tooltip container
-			const parent = element.parentElement;
-			container = document.createElement( 'div' );
-			container.classList.add( 'highlight-tooltip__container' );
-			parent.appendChild( container );
-			setNode( container );
+			if ( ! useAnchor ) {
+				const parent = element.parentElement;
+				container = document.createElement( 'div' );
+				container.classList.add( 'highlight-tooltip__container' );
+				parent.appendChild( container );
+				setNode( container );
+			} else {
+				const parentElement = document.createElement( 'div' );
+				document.body.appendChild( parentElement );
+				container = document.createElement( 'div' );
+				parentElement.appendChild( container );
+				setNode( container );
+				setAnchorRect( element.getBoundingClientRect() );
```

Comment:
When enabling `useAnchor` we append the tooltip to a div tag in the `body` and pass the `anchorRect` of the id referenced to the Popover element.

Reply:
This works perfectly for our use case!

The only thing I notice is that sometimes the position is not correct on the first load (presumably the targeted element is still mounting).  One possible workaround is updating this when `showHighlight` changes.

```js
	useEffect( () => {
		setAnchorRect( element.getBoundingClientRect() );
	}, [ showHighlight ] );
```